src/operator/quantization/quantized_conv.cc

```diff
@@ -0,0 +1,171 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2017 by Contributors
+ * \file quantized_conv.cc
+ * \brief
+ * \author Ziheng Jiang, Jun Wu
+*/
+#include "../nn/convolution-inl.h"
+
+namespace mxnet {
+namespace op {
+
+// TODO(junwu): Reuse the InferShape function of convolution op after
+// this pr is merged: https://github.com/apache/incubator-mxnet/pull/8302
```

Comment:
That PR has been merged

Reply:
Just tried. That function for FP32 convolution seems not implemented appropriately. The data member `layout` of struct `ConvolutionParam` is a `option<int>` type, but it was used with the assumption that `layout` contains a real value in that function. This will cause my test case to fail. Making that function properly implemented needs another PR. For now, I will just keep the shape inference for quantized_conv op. Already deleted the TODO comment.